doc/api/resources/sendmail_rules.rst

```diff
@@ -27,6 +27,10 @@ restrict_to_status                    list                       List of order s
                                                                  ``n__valid_if_pending`` for payment pending but already confirmed,
                                                                  and ``n__pending_overdue`` for pending with payment overdue.
                                                                  The default is ``["p", "n__valid_if_pending"]``.
+checked_in_status                     string                     Checked in status to restrict recipients to. Valid strings
+                                                                 are: ``all`` for no filtering (default), ``checked_in`` for limiting to
```

Comment:
I thought about that too, although I thought of `"ignore"` instead of `null`. I do like `null` better though – this is more similar to just leaving the query-param out completely (which works as well).

Reply:
This was a bigger change than anticipated, but I agree with you both and changed it to accept `null`